File: Python/Program/Pengiriman.py
from Pembelian import Pembelian

# Kelas Pengiriman turunan dari Pembelian
class Pengiriman(Pembelian):
    def __init__(self):
        super().__init__()
        self.alamat = ""               # Alamat pengiriman
        self.ekspedisi = ""            # Nama ekspedisi
        self.waktu_pengiriman = ""     # Estimasi waktu pengiriman

    def set_alamat(self, alamat):
        self.alamat = alamat           # Setter alamat pengiriman

    def get_alamat(self):
        return self.alamat             # Getter alamat pengiriman

    def set_ekspedisi(self, ekspedisi):
        self.ekspedisi = ekspedisi     # Setter nama ekspedisi

    def get_ekspedisi(self):
        return self.ekspedisi          # Getter nama ekspedisi

    def set_waktu_pengiriman(self, waktu):
        self.waktu_pengiriman = waktu  # Setter estimasi waktu pengiriman

    def get_waktu_pengiriman(self):
        return self.waktu_pengiriman   # Getter estimasi waktu pengiriman

File: Java/Program/Pengiriman.java
public class Pengiriman extends Pembelian {
    private String alamat;              // Alamat pengiriman
    private String ekspedisi;           // Nama ekspedisi
    private String waktuPengiriman;     // Estimasi waktu pengiriman

    public Pengiriman() {}              // Konstruktor default

    // Setter alamat pengiriman
    public void setAlamat(String alamat) {
        this.alamat = alamat;
    }

    // Getter alamat pengiriman
    public String getAlamat() {
        return alamat;
    }

    // Setter ekspedisi
    public void setEkspedisi(String ekspedisi) {
        this.ekspedisi = ekspedisi;
    }

    // Getter ekspedisi
    public String getEkspedisi() {
        return ekspedisi;
    }

    // Setter estimasi waktu pengiriman
    public void setWaktuPengiriman(String waktu) {
        this.waktuPengiriman = waktu;
    }

    // Getter estimasi waktu pengiriman
    public String getWaktuPengiriman() {
        return waktuPengiriman;
    }
}
